floodviz/views.py

```diff
@@ -40,8 +40,8 @@ def timeseries_data():
 def _peakflow_helper():
     # Peak Flow config vars #
     peak_site = app.config['PEAK_SITE']
-    peak_start_date = app.config['PEAK_START_DT']
-    peak_end_date = app.config['PEAK_END_DT']
+    peak_start_date = app.config['EVENT_START_DT']
```

Comment:
Fine for now. At some point, we should remove the start_date from the peak_utils.req_peak_data function as it is not used.